features/provider/dashboard/tabs.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Dashboard search bar
+  In order to navigate easily to products and backends
+  As a provider
+  I want to be able to filter them by name
+
+  Background:
+    Given a provider is logged in
+    And a default service of provider "foo.example.com" has name "My Fancy Product"
+    And a service "My Regular Product" of provider "foo.example.com"
+    And a backend api "My Fancy Backend API" of provider "foo.example.com"
+    And a backend api "My Regular Backend API" of provider "foo.example.com"
```

Comment:
If you follow [this](https://github.com/3scale/porta/pull/1406/files#r344196825), then this part would be:
```
And default service has name "My Fancy Product"
And a service "My Regular Product"
And a backend api "My Fancy Backend API"
And a backend api "My Regular Backend API"
```